handel/src/aws/pricing-calls.ts

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2018 Brigham Young University
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+import awsWrapper from './aws-wrapper';
+
+function convertMemoryStringToMemoryUnits(inputString: string) {
+    // Converts the given GiB into the amazon units
+    const gibiBitesOfMemory = inputString.split('GiB', 1);
+    const amazonMemoryUnits = Number(gibiBitesOfMemory) * 1000;
+    return amazonMemoryUnits;
+}
+
+export async function getMemoryForInstance(instanceType: string): Promise<number> {
+    const getProductsParams = {
+        ServiceCode: 'AmazonEC2',
+        Filters: [
+            {
+                Type: 'TERM_MATCH',
+                Field: 'instanceType',
+                Value: instanceType
+            },
+            {
+                Type: 'TERM_MATCH',
+                Field: 'location',
+                Value: 'US West (Oregon)'
```

Comment:
This needs to be changed to support any region, since Handel can deploy to any region. We'll probably need a mapping from region names like 'us-west-2' to the value they want like 'US West (Oregon)'.

Better yet, we should check the API for GetProducts agains to see if there is a filter that will let us give the actual region name like 'us-west-2'.